packages/markdown/remark/src/index.ts

```diff
@@ -61,7 +62,6 @@ export async function renderMarkdown(
 		frontmatter: userFrontmatter = {},
 	} = opts;
 	const input = new VFile({ value: content, path: fileURL });
-	const scopedClassName = opts.$?.scopedClassName;
```

Comment:
I removed this since it seems to only been used for Astro-flavoured markdown, which we don't use anymore.